python/example_code/apigateway/aws_service/setup.yaml

```diff
@@ -0,0 +1,124 @@
+Resources:
```

Comment:
Can we / should we add a copyright header?

Reply:
I think no, because the file is generated and so we'd have to re-add the copyright every time.